src/Game.java

```diff
@@ -1,13 +1,64 @@
-import java.util.ArrayList;
+import java.util.Date;
+import java.util.Random;
+import java.util.Scanner;
 
 public class Game {
-  static ArrayList<Zombie> zombies = new ArrayList<Zombie>();
   static Board board = new Board(4, 4);
 
+  private static void printOptions() {
+    System.out.println("1: Place sunflower");
+    System.out.println("2: Place peashooter");
+    System.out.println("3: Skip Turn");
+    System.out.println("4: Quit");
+    System.out.print("Enter option number of what you would like to do: ");
+  }
+
+  private static PieceLocation parseCoordinates(String input){
+    //split coordinates by comma and remove any spaces
+    String[] locationSplit = input.split(",");
+    int row = Integer.parseInt(locationSplit[0].trim());
+    int col = Integer.parseInt(locationSplit[1].trim());
+    if(row > 3 || col > 3){
```

Comment:
row and col shouldnt be hard coded, getNumRows() or something

Reply:
I can add `getNumRows()` and `getNumCols()` in the board class.

Reply:
added in my branch